build_resolvers/lib/src/resolver.dart

```diff
@@ -134,7 +137,9 @@ class AnalyzerResolver implements ReleasableResolver {
     var path = assetPath(assetId);
     var uri = assetId.uri;
     var source = _driver.sourceFactory.forUri2(uri);
-    if (source == null) throw ArgumentError('missing source for $uri');
+    if (source == null || !source.exists()) {
+      throw ArgumentError('missing source for $uri');
```

Comment:
[out of scope?] I wonder if this should have been `AssetNotFoundException`... Is there any risk in changing it?

Reply:
I will go ahead and change it, there is some small risk but it should be very minor.